README.md

```diff
@@ -25,9 +25,14 @@ Thanks to the awesome folks over at [Fastly][fastly], there's a free, CDN hosted
 <link href="//vjs.zencdn.net/5.19/video-js.min.css" rel="stylesheet">
 <script src="//vjs.zencdn.net/5.19/video.min.js"></script>
 ```
-
 > For the latest version of video.js and URLs to use, check out the [Getting Started][getting-started] page on our website.
 
+> We include a [stripped down Google Analytics pixel](https://github.com/videojs/cdn/blob/master/src/analytics.js) that tracks a random sampling (currently 1%) of players loaded from the CDN. This allows us to see (roughly) what browsers are in use in the wild, along with other useful metrics such as OS and device. If you'd like to disable analytics, you can simply include the following global before including Video.js via the free CDN:
```

Comment:
Just to be extra explicit that this is the CDN version how about "Only the CDN version of Video.js includes this analytics pixel. If you'd like to use the CDN hosted version but would like to disable analytics, you can simply..."